### DISCUSSION TITLE
Passing configuration to the app

### DISCUSSION THREAD

**Discussion by archiif** [Textualize/textual#353] [Q&A] [answered]
I want to pass user configuration when initiating an `App` instance, what is the idiomatic way of doing this? I tried doing something like this:

```python
class TuiApp(App):
    def __init__(self, *args, value: str, **kwargs):
        self.value = value
        super().__init__(*args, **kwargs)
```
But I end up getting this error:
```
TypeError: TuiApp.__init__() missing 1 required keyword-only argument: 'value'
```

**Comment by ThatXliner** [comment, marked as answer]
Maybe https://github.com/Textualize/textual/issues/70 would help
